CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ### Added
 
 - Add the new `go.opentelemetry.io/contrib/instrgen` package to provide auto-generated source code instrumentation. (#3068, #3108)
+- Add support for `cloud.{account.id,availability_zone,region}` in the AWS ECS detector
```

Comment:
- please break these out into full words so users can search the changelog for them
- please add a period at the end of the sentence to follow existing formatting
- please add the PR number to follow existing formatting

Reply:
Done in 3b8608d4 and 0eca55d7